instructions.md

```diff
@@ -0,0 +1,140 @@
+
+You could put these in your bashrc so they are automatically in your environment
+when you login.
+
+3. Basic HIL commands:
+
+* `hil list_project_nodes <project-name>`
+    This will list all the nodes that your project has access to.
+
+* `hil show_node <node-name>`
+    This will show you the information about your node.
+
+* `hil node_connect_network <node-name> <nic-name> <network-name> <channel>`
+    to connect your node's nic to a network. Channel could either be `vlan/native`
+    or `vlan/<vlan-id>`.
+
+* to find the vlan-id of a network, run `hil show_network <network-name>`
+
+* `hil node_detach_network <node-name> <nic-name> <network-name>`
+    this will remove <network-name> from your node's nic.
+
+* `hil node_power_cycle <node-name>`.
+    to power cycle a node.
+
+* `hil node_power_off <node-name>`.
+    to power off a node.
+
+
+## Using BMI
+
+1. Logon to the BMI machine (available only through the HIL client).
+    `ssh username@kumo-bmi-no-seccloud.infra.massopen.cloud`
+    or use it's ip address which is 192.168.100.142
+
+2. Once logged in, export your HIL credentials explained above. In addition to that
+do this:
+    `export BMI_CONFIG=/etc/bmi/bmiconfig.cfg`
+
+Test bmi works by running `bmi db ls` command.
+
+
+3. Run this command to see a list of available images. The project name is the same name as in HIL.
+    `bmi ls <project-name>`
+
+
+4. If this is the first time you got a node, and you want to provision it with an OS using BMI.
+Run this command:
+    `bmi provision <project-name> <node-name> <image-name> bmi-provision-net-no-seccloud em1`
+
+
+    * Here bmi-provision-net-no-seccloud is the name of the provisioning network,
+    and em1 is the name of the nic. You could change it if you know what you are doing.
+
+    * Make sure that the network `bmi-provision-net-no-seccloud` is not already connected.
+    If it is, then disconnect that network using HIL and then run the BMI command again.
+
+    * The command should return `success` within 5-10 seconds.
+
+
+If you had already provisioned a node using bmi, and just want to reconnect your image
+to your node, run this script available in your path.
+reconnect.sh <project-name> <node-name>
+
+    This command will only work if you already had that node provisioned using bmi.
+
+
+5. Once you provision your node, power cycle your node using HIL and it should boot
+into your image in 3-4 minutes.
+
+For dell-XX node, the ip address is 10.10.10.1XX
+eg; dell-4 = 10.10.10.104, dell=12 = 10.10.10.112
+
+You can now ssh to your nodes using those IP addresses.
+
+
+6. To deprovision a node, run this:
+    `bmi dpro <project-name> <node-name> bmi-provision-net-no-seccloud em1`
+
+    * Make sure that the network bmi-provision-net-no-seccloud is connected to the node
+    you are trying to deprovision. If not, attach is natively using HIL.
+
+    * This command should return `success` within 5-10 seconds. This will **delete** your
+    image.
+
+
+## How to SSH
+
```

Comment:
I think it should be a different README file, or at least be mentioned at the beginning of the document that they can find more information about ssh at the end of this document .

Reply:
This is just a reference because a lot of people keep asking me. I didn't intend this to be a formal document, but rather one with helpful information. I pushed it the bottom because, like you said, it's kind of different.